src/helpers.js

```diff
@@ -1,3 +1,17 @@
+function isNulledBox(box) {
+  return !box.w && !box.h && !box.x && !box.y
+}
+
+function domContains(node) {
```

Comment:
especially this helper would look really ugly when inlined. As I already said: We dont have these helpers for the user in the first place. They are there to do recurring tasks. Making them public is just a plus.